exercises/atbash-cipher/example.ml

```diff
@@ -0,0 +1,57 @@
+let substitute = function
```

Comment:
You are not using Core here. Doing so would mean you could re-use their implementations of a few functions (```drop``` and ```take``` for example).

Is it a deliberate decision though - perhaps to provide a measure of how difficult the exercise is to implement using the most basic OCaml enviroment???